packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -239,15 +238,13 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
               </li>
             ))}
           </ul>
-          {!isSecondary && (
             <button
-              className={css(styles.tabsScrollButton)}
+              className={css(styles.tabsScrollButton, isSecondary && buttonStyles.modifiers.secondary)}
               aria-label={rightScrollAriaLabel}
```

Comment:
This reads "Scroll Right" but should be "Scroll right" (sentence case), and match `leftScrollAriaLabel`.

Reply:
Ok, nice catch! :)